code/game/machinery/iv_drip.dm

```diff
@@ -162,10 +162,10 @@
 	. = ..()
 	if(.)
 		return
-	if(!ishuman(user))
+	if(!ishuman(user) && !iscyborg(user))
```

Comment:
Do cyborgs even call attack_hand?

Reply:
[Yes](https://github.com/yogstation13/Yogstation/blob/7115e36f9db7d13416c3152be3c0c8184c356c93/code/game/machinery/iv_drip.dm#L179). It seemed better to do that instead of copying and pasting all of attack_hand into attack_robot.